zerver/webhooks/uptimerobot/doc.md

```diff
@@ -1,33 +1,47 @@
+# Zulip UptimeRobot integration
+
 Receive Zulip notifications from UptimeRobot!
 
+{start_tabs}
+
 1. {!create-channel.md!}
 
 1. {!create-an-incoming-webhook.md!}
 
-1. {!generate-integration-url.md!}
+1. {!generate-webhook-url-basic.md!}
 
-1. On UptimeRobot, go to **My Settings** and select **Add Alert Contact**.
-   Set **Alert Contact Type** to **webhook**.
+1. On UptimeRobot, go to **My Settings** and select **Add Alert Contact**.<br>
+   Set **Alert Contact Type** to **webhook**.  
+   Set **Friendly Name** to a name of your choice, such as `Zulip`.  
+   Set **URL to notify** to the URL generated above.  
 
-1. Set **Friendly Name** to a name of your choice, such as `Zulip`. Set **URL to notify** to the
-   URL constructed above. Under **POST Value (JSON Format)**, select **Send as JSON (application/json)**.
+1. Under **POST Value (JSON Format)**, select **Send as JSON (application/json)**.<br>
+   Set **POST Value(JSON Format)** to:
 
-1. Set **POST Value(JSON Format)** to:
+         {
+            "monitor_url":"*monitorURL*",
+            "monitor_friendly_name":"*monitorFriendlyName*",
+            "alert_type":"*alertType*",
+            "alert_type_friendly_name":"*alertTypeFriendlyName*",
+            "alert_details":"*alertDetails*",
+            "alert_friendly_duration":"*alertFriendlyDuration*"
+         }
 
-      ```
-      {
-         "monitor_url":"*monitorURL*",
-         "monitor_friendly_name":"*monitorFriendlyName*",
-         "alert_type":"*alertType*",
-         "alert_type_friendly_name":"*alertTypeFriendlyName*",
-         "alert_details":"*alertDetails*",
-         "alert_friendly_duration":"*alertFriendlyDuration*"
-      }
-      ```
+1. Set **Enable Notifications For** the events you'd like to be notified
+   about. Save the form.
```

Comment:
Let's link to the events section and make this one sentence, e.g. "notified about, and save the form."